build.gradle

```diff
@@ -22,6 +22,10 @@ dependencies {
 
 	//Test Dependencies	
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	
+	// LocalDate etc.
+	runtimeOnly 'com.fasterxml.jackson.module:jackson-modules-java8'
+
```

Comment:
I don't know why but the build will be failed at my end if not adding the version (2.12.4) of "jackson-modules-java8" module.
testImplementation 'com.fasterxml.jackson.module:jackson-modules-java8:2.12.4'
does anyone get this issue?